1_fetch/src/1_MendotaPull.R

```diff
@@ -0,0 +1,20 @@
+## This script pulls down model evaluation data for Lake Mendota
+
+## Function for pulling down data from 
+## USGS's ScienceBase https://www.sciencebase.gov/
+
+fetch_data <- function(ID, project_output_dir, file_name){
+  require(sbtools)
+  
+  if(dir.exists(project_output_dir) == F){
+    dir.create(project_output_dir)
+  }
+  
+  path <- file.path(project_output_dir, file_name)
+  item_file_download(ID, names = file_name, destinations = path, overwrite_file = TRUE)
+  
+  print(paste0('This file has been saved as ', path ))
```

Comment:
Printed-out things become hard to see / aren't visible at the right times with big targets pipelines. But I like that you're thinking about how to communicate this important information - **targets** has a different solution that you'll see soon.